tests/system/Bigtable/ReadRowsAcceptanceTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\System\Bigtable;
+
+require_once __DIR__ .'/../../../vendor/autoload.php';
+
+use Google\ApiCore\ServerStream;
+use Google\ApiCore\ValidationException;
+use Google\ApiCore\Testing\MockServerStreamingCall;
```

Comment:
Should we be using mocks in system tests?

Reply:
This is a unit test, and should be moved to the unit test directory.